test/server/cards/01_SOR/units/FirstLegionSnowTrooper.spec.js

```diff
@@ -0,0 +1,86 @@
+describe('First Legion Snow Trooper', function() {
+    integration(function() {
+        describe('First Legion Snow Trooper while attacking non-damaged unit.', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['first-legion-snowtrooper'],
+                    },
+                    player2: {
+                        groundArena: ['yoda#old-master'],
+                    }
+                });
+            });
+
+            it(' Ability should do nothing', function () {
+                this.player1.clickCard(this.firstLegionSnowtrooper);
+                this.player1.clickCard(this.yodaOldMaster);
+                expect(this.firstLegionSnowtrooper.exhausted).toBe(true);
+                expect(this.firstLegionSnowtrooper.damage).toBe(2);
+                expect(this.yodaOldMaster.damage).toBe(2);
+                expect(this.player2).toBeActivePlayer();
+            });
+        });
+        describe('First Legion Snow Trooper with a damaged enemy unit.', function() {
```

Comment:
Do you mind adding a newline before this? For some reason the linter doesn't catch this case, I need to investigate

Reply:
Added the newline.